.releaserc.json

```diff
@@ -0,0 +1,44 @@
+{
+    "branches": [
+      "main"
+    ],
+    "ci": false,
+    "plugins": [
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "conventionalcommits"
+        }
+      ],
+      [
+        "@semantic-release/release-notes-generator",
+        {
+          "preset": "conventionalcommits"
+        }
+      ],
+      [
+        "@semantic-release/github",
+        {
+          "successComment": "This ${issue.pull_request ? 'PR is included' : 'issue has been resolved'} in version ${nextRelease.version} :tada:",
+          "labels": false,
+          "releasedLabels": false
+        }
+      ],
+      [
+        "@semantic-release/changelog",
+        {
+          "changelogFile": "CHANGELOG.md",
+          "changelogTitle": "# Changelog\n\nAll notable changes to this project will be documented in this file."
+        }
+      ],
+      [
+        "@semantic-release/git",
```

Comment:
The default `GITHUB_TOKEN` provided by GitHub Actions can't push to protected branches.  Even if we used a deploy key or a personal access token (PAT) to get around that limitation, the branch protection rules that are generally in place in this GitHub org will block pushing even with a deploy key or PAT.  It is possible for someone with admin permissions to temporarily, manually unblock that, but we should make extremely limited use of that behavior.

@displague has previously suggested that we can use the releases page as our changelog, and if we do have a `CHANGELOG.md`, it can just have some text and a link pointing to the releases page as the place where changes for each release are documented.

Reply:
Agreed. Based on that, we'd remove these two changelog steps and update the CHANGELOG.md in the project to link to GH Releases.

Reply:
Thank you! Release job no longer updates the CHANGELOG.md file in [4610d02](https://github.com/equinix-labs/terraform-equinix-labs/pull/16/commits/4610d026194a8437aa91b7e8979ab48eda44b3ff).

See tests here:
- [release tags] (https://github.com/dcallao/terraform-equinix-labs-test/releases/tag/v1.1.0)
- [updates in CHANGELOG.md](https://github.com/dcallao/terraform-equinix-labs-test/blob/main/CHANGELOG.md)